src/cplscheme/tests/ParallelImplicitCouplingSchemeTest.cpp

```diff
@@ -14,7 +14,7 @@
 #include "cplscheme/ParallelCouplingScheme.hpp"
 #include "cplscheme/SharedPointer.hpp"
 #include "cplscheme/config/CouplingSchemeConfiguration.hpp"
-#include "cplscheme/impl/MinIterationConvergenceMeasure.hpp"
+#include "cplscheme/impl/AbsoluteConvergenceMeasure.hpp"
```

Comment:
Where does this come from? Does not really look like it belongs to this PR. Maybe consider just adding `#include "cplscheme/impl/AbsoluteConvergenceMeasure.hpp"` directly on `develop` to clean up the diff a bit.

Reply:
Oh. I used a nonsense absolute measure before adding support for no conv measures.